doc/fta-construction-algorithm.md

```diff
@@ -39,11 +39,24 @@ BEGIN
 END
 ```
 
-## Example 1
+## Example 1 - Or Behaviour
 Demonstrates transformation of OR behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree](https://user-images.githubusercontent.com/18463762/134146813-8a257931-26bb-4350-b954-c29ac7601c9c.png)
 
-## Example 2
+## Example 2 - And Behaviour
 Demonstrates transformation of AND behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree-example-2](https://user-images.githubusercontent.com/18463762/137472141-065769d1-8f6c-4267-ac5e-4fc85c16a0c4.png)
 
+## Example 3 - Live Demo
+This example is a study by [1]. The analyzed system (as shown in the follwing picture) is aircraft fuel system, including its subsystems (fuel transportation, fuel supplying and monitoring subsystem).  
```

Comment:
@ahmadjana we should not only say that the example is from study, but also that the referred picture is form it, unless we draw it.

Reply:
done

Reply:
https://github.com/kbss-cvut/fta-fmea-ui/pull/39/commits/9b420d35b95f9d934c5a7228778a8bc889c1c921

resolve if agree @ahmadjana

Reply:
@ahmadjana